src/parse_logcfg.h

```diff
@@ -43,7 +43,6 @@ extern char *error_details;
 
 ////////////////////////////////////
 // config parsing definitions
-
```

Comment:
please revert this file (and main.c) as there were not changed.

Reply:
Reverted